docs/core_docs/docs/integrations/retrievers/azion-edgesql.mdx

```diff
@@ -0,0 +1,53 @@
+### Azion Edge SQL Retriever
```

Comment:
Can we have this use the standard retriever template?

https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-scripts/src/cli/docs/templates/retrievers.ipynb